components/inputs/input-date.js

```diff
@@ -191,12 +192,15 @@ class InputDate extends LocalizeStaticMixin(LitElement) {
 	}
 
 	render() {
-		const shortDateFormat = (this._dateTimeDescriptor.formats.dateFormats.short).toUpperCase();
+		const formattedWideDate = formatISODateInUserCalDescriptor('2020-12-20');
```

Comment:
Fixes iOS width being too small

Reply:
Ah so like the numbers can be wider than the characters in the format descriptor - crazy! Is there one number that's the widest? Like "3"? i.e. maybe `2323-12-23` might be wider.

Reply:
Yeah so in the case of english the placeholder is M/D/YYYY so just having more characters I think causes the numbers to be wider. In Lato it looks like all numbers are the same width, but it could be in other languages with different fonts that that isn't true so I like the idea of using 3 - it seems wider.